infra/cifuzz/cifuzz.py

```diff
@@ -331,35 +330,35 @@ def get_files_covered_by_target(latest_cov_info, target_name, src_in_docker):
     return None
 
   # Make sure cases like /src/curl and /src/curl/ are both handled.
-  norm_src_in_docker = os.path.normpath(src_in_docker)
-  if not norm_src_in_docker.endswith('/'):
-    norm_src_in_docker += '/'
+  norm_oss_fuzz_repo_path = os.path.normpath(oss_fuzz_repo_path)
+  if not norm_oss_fuzz_repo_path.endswith('/'):
```

Comment:
I'm curious why this is necessary to end it with a slash?

Reply:
If there is a directory in $SRC that starts with the same name as the repo, then `if not norm_file_path.startswith(norm_oss_fuzz_repo_path):` will trigger. For example curl has both a /curl and /curl_fuzzers. But we want to ignore /curl_fuzzers so I add the `/`